TFT/src/User/API/coordinate.c

```diff
@@ -69,6 +70,18 @@ void coordinateSetAxisTarget(AXIS axis, float position)
   }
 }
 
+// Set level corner position the measured Z offset from probe, see in ABL.c menu refreshLevelCornerValue(MENUITEMS levelItems) and value get from parseACK.c
```

Comment:
move your new two functions after coordinateQuery so you will respect the same order in the .h file